finalProject/tenSpeckledFrogs.js

```diff
@@ -0,0 +1,11 @@
+for (var i = 1; i < 11; i++){
+  if (i == 1){
```

Comment:
Right now, this loop is set up so that i is assigned to 1 for the first loop, so that first conditional would be the  first to process. You'll need to adjust your conditionals here.